### DISCUSSION TITLE
3ds cia roms -

### DISCUSSION THREAD

**Discussion by unabletoconnect2429** [retronas/discussions#58] [Q&A]
Should the cia directory for the qr generater be a symbolic link to the roms/nintendo/3ds directory?

Assuming that its still a pending an update from before the roms folder was implemented.

**Comment by danmons** [comment]
Might be a good idea.  I've been pondering different options inside the generic `roms` dir to try and satisfy different tools where they only want one type of format (OpenPS2Loader, for example, which separates CD and DVD ISOs).

I might create a sub `cia` and `3ds` folder below the current roms/nintendo/3ds, and then point the QR code `cia` folder to the generic `roms` dir.

**Reply by sairuk** [reply]
this is now in testing, /3ds/cia is now a symlink to <generic>/roms/nintendo/3ds
